tests/app/config/packages/neusta_converter.yaml

```diff
@@ -0,0 +1,16 @@
+neusta_converter:
+  converter:
+    test.person.converter:
+      converter: Neusta\ConverterBundle\Converter\DefaultConverter
+      target_factory: Neusta\ConverterBundle\Tests\Fixtures\Factory\PersonFactory
+      populators:
+        - Neusta\ConverterBundle\Tests\Fixtures\Populator\PersonNamePopulator
+
+    test.person.converter.with.cache:
+      converter: Neusta\ConverterBundle\Converter\DefaultConverter
+      target_factory: Neusta\ConverterBundle\Tests\Fixtures\Factory\PersonFactory
+      populators:
+        - Neusta\ConverterBundle\Tests\Fixtures\Populator\PersonNamePopulator
+      cached:
+#        service: user.cache_management
```

Comment:
This commented line of configuration means: you can set cache management service but if you use the default one, skip it?!

Reply:
The config currently allows defining a custom cache `CacheManagement` implementation via the `service` key *or* only a `CacheKeyFactory` service (if you want to use the `DefaultCacheManagement`) via the `key_factory` key.